st2api/st2api/wsgi.py

```diff
@@ -18,7 +18,7 @@
 
 from st2api import config  # noqa
 from st2common import log as logging
-from st2common.models import db
+from st2common.persistence import db_init
```

Comment:
This is going to be confusing. persistence seems like the right place though. sigh.